launcher/meta/BaseEntity.cpp

```diff
@@ -97,9 +104,10 @@ bool Meta::BaseEntity::loadLocalFile()
     // TODO: check if the file has the expected checksum
     try
     {
-        auto doc = Json::requireDocument(fname, fname);
-        auto obj = Json::requireObject(doc, fname);
-        parse(obj);
+        //auto doc = Json::requireDocument(fname, fname);
+        //auto obj = Json::requireObject(doc, fname);
+        const auto json = nlohmann::json::parse(std::ifstream(fname.toStdString()));
+        parse(json);
         return true;
     }
     catch (const Exception &e)
```

Comment:
I believe you need to catch nlohmann::json::parse_error here

Reply:
yes, there are probably some more that I've missed/have used std::exception lazily, will get to them